argon2/src/lib.rs

```diff
@@ -53,6 +53,7 @@
 //!
 //! // Verify password against PHC string
 //! let parsed_hash = PasswordHash::new(&password_hash)?;
+//! Note hash params from `parsed_hash` are used instead of what is configured in `argon2` here !
```

Comment:
This is failing to parse because it needs to be a nested comment.

```suggestion
//! // Note hash params from `parsed_hash` are used instead of what is configured in `argon2` here !
```